zen/pkm/storage.py

```
@@ -0,0 +1,344 @@
+    def load_knowledge_entry(self, entry_id: str) -> Optional[KnowledgeEntry]:
+        """Load a knowledge entry by ID."""
+        json_path = self.knowledge_base_dir / f"{entry_id}.json"
+        
+        if not json_path.exists():
+            return None
+        
+        try:
+            with open(json_path, 'r', encoding='utf-8') as f:
+                data = json.load(f)
+            return KnowledgeEntry(**data)
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)